Dockerfile

```diff
@@ -53,3 +52,35 @@ COPY extend-environment.sh /etc/profile.d/extend-environment.sh
 RUN pip install --no-cache-dir -r requirements/local.txt
 
 CMD ["python", "./cfgov/manage.py", "runserver", "0.0.0.0:8000"]
+
+FROM cfgov-develop as cfgov-build
+
+# add node and yarn repos
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
+
+RUN yum install -y nodejs yarn  && \
```

Comment:
I'd say we're okay leaving this how it is, I agree 10.x is fine and the way the yarn repo is added above, you get `stable` by default, which is probbbbbbably fine for a command line tool.

In a production image, though, I'd think we'd want yarn and node's install in a separate stage from from the actual frontend build, so that it wouldn't be rerun on every code change

Reply:
Does that really matter, though?  Yes, the `COPY`s and forward would rerun in this stage, but the expensive operations (nodejs/yarn install here) would remain cached.

Reply:
Ahhh, that's a good point! Yeah so I guess this should work well then going forward.